packages/partywhen/src/index.ts

```diff
@@ -4,7 +4,7 @@ import { Server } from "partyserver";
 export type RawTask = {
   id: string;
   description?: string | undefined;
-  payload?: Record<string, unknown> | undefined;
+  payload?: Record<string, Rpc.Serializable<unknown>> | undefined;
```

Comment:
Do we really need `Record` here? Maybe this should just be `Rpc.Serializable<unknown>` ?